cms/static/js/views/xblock_outline.js

```diff
@@ -322,7 +322,7 @@ define(['jquery', 'underscore', 'gettext', 'js/views/baseview', 'common/js/compo
              */
             handleDuplicateEvent: function(event) {
                 var self = this,
-                    xblockType = XBlockViewUtils.getXBlockType(self.model.get('category'), self.parentView.model, true),
+                    xblockType = XBlockViewUtils.getXBlockType(self.model.get('category'), self.parentView.model),
```

Comment:
@Qubad786 Because of this: 
```
 var xblockMap = {
                         unit: 'subsection',
                         subsection: 'section',
                         section: 'course'
                     },
                     parentXblockType = xblockMap[xblockType];
```
Here xblockType is either 'section/subsection/unit' which maps in the Xblock. thats why translation is not necessary.